debian/rules

```diff
@@ -222,13 +222,17 @@ apt: build-binary build-manpages debian/apt.install
 	#   debian/$@/usr/lib/apt/apt-report-mirror-failure \
 
 	# move the apt-helper in place
-	mv debian/$@/usr/bin/apt-helper debian/$@/usr/lib/apt/apt-helper 
+	mv debian/$@/usr/bin/apt-helper debian/$@/usr/lib/apt/apt-helper
+	# install apt.systemd.daily helper in the right place
+	install -m755 debian/apt.systemd.daily debian/$@/usr/lib/apt/
 
 	dh_bugfiles -p$@
 	dh_lintian -p$@
 	dh_installexamples -p$@ $(BLD)/docs/examples/*
 	dh_installman -p$@ $(wildcard $(patsubst %,doc/en/%.[158],$(apt_MANPAGES)) $(patsubst %,doc/*/%.*.[158],$(apt_MANPAGES)))
-	dh_installcron -p$@
+	dh_installcron -p$@ --name=apt-compat
+	dh_systemd_enable -p$@ apt-daily.timer apt-daily.service 
```

Comment:
The apt-daily.service argument is a noop because it does not have an [Install] section